internal/sync/cache.go

```diff
@@ -471,12 +473,17 @@ type ResourceType struct {
 }
 
 func (n *ResourceFilter) IsExcludedResource(group, kind, _ string) bool {
-	for _, resource := range n.includedResources {
-		// When Kind is empty, we only check if Group matches
+	for _, resource := range n.IncludedResources {
 		if resource.Kind == "" {
+			// When Kind is empty, we only check if Group matches
 			if group == resource.Group {
 				return false
 			}
+		} else if resource.Group == "" {
+			// When Group is empty, we only check if Kind matches
+			if group == resource.Kind {
+				return false
+			}
```

Comment:
I wonder if we need this. Maybe we shouldn't encourage users to only specify `Kind` and not `Group` since the reason `Group` exists is to serve as a "namespace" which can distinguish Kinds that happen to have the same name.

Reply:
Did you have an issue of specifying ConfigMap that it didn't work? I think that may have `group==""`?

Reply:
I was previously seeing an error adding `Secret` to the controller configmap but it was a formatting issue instead of a code error. Just tested this out and it is back to working like we intend. In this case, I'll remove this extra clause I added.